### PR TITLE
[frameworks][now-static-build] Support Docusaurus v2 and v1

### PR DESCRIPTION
**Pull request by AndyBitz**
Support Docusaurus v2 and v1

### CODE REVIEW

File: packages/frameworks/frameworks.json [hunk 1]
```diff
@@ -143,6 +139,34 @@
       }
     }
   },
+  {
+    "name": "Docusaurus",
+    "slug": "docusaurus",
```
Comment (styfle) [inline packages/frameworks/frameworks.json:144]:
```suggestion
    "slug": "docusaurus-1",
```

Perhaps we should be specific here?

Comment (AndyBitz) [reply packages/frameworks/frameworks.json:144]: @styfle no, that would break existing configs

File: packages/frameworks/frameworks.json [hunk 2]
```diff
@@ -112,19 +112,15 @@
...
   {
-    "name": "Docusaurus",
-    "slug": "docusaurus",
+    "name": "Docusaurus 2",
+    "slug": "docusaurus-2",
     "demo": "https://docusaurus.now-examples.now.sh",
     "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/docusaurus.svg",
     "tagline": "Docusaurus makes it easy to maintain Open Source documentation websites.",
     "description": "A static Docusaurus site that makes it easy to maintain OSS documentation.",
     "website": "https://docusaurus.io/",
```
Comment (styfle) [inline packages/frameworks/frameworks.json]:
```suggestion
    "website": "https://v2.docusaurus.io",
```

File: packages/frameworks/frameworks.json [hunk 3]
```diff
@@ -111,6 +111,34 @@
...
+      "devCommand": {
+        "value": "docusaurus start --port $PORT"
+      },
+      "outputDirectory": {
+        "value": "build"
+      }
+    }
+  },
   {
     "name": "Docusaurus",
```
Comment (styfle) [inline packages/frameworks/frameworks.json]:
```suggestion
    "name": "Docusaurus 1",
```

Is this the name that shows up in the dropdown?

File: packages/now-static-build/src/frameworks.ts [hunk 4]
```diff
@@ -80,7 +80,7 @@ const frameworkList: Framework[] = [
   },
   {
     name: 'Docusaurus',
```
Comment (styfle) [inline packages/now-static-build/src/frameworks.ts]:
```suggestion
    name: 'Docusaurus 2',
```

Lets make this consistent with frameworks.json

File: packages/frameworks/frameworks.json (hunk 3 above)
Comment (AndyBitz) [reply packages/frameworks/frameworks.json]: yes

**Review by styfle — APPROVED** [review]
Perfect, thanks! 👌

**Pull request merged**